### ISSUE TITLE
MEMBERSHIP heading text isn't responsive

### ISSUE THREAD

**Issue by tikhsuP** [fossasia/fossasia.org#686]
### Description

The word "MEMBERSHIP" in the
`<div class="col-sm-12"><span class="price">Membership</span></div>`

overlaps the text paragraph next to it for the screen sizes from 976px to 1184px.

### Screenshot
![Screenshot (34)](https://user-images.githubusercontent.com/42881874/77820641-babfd580-7109-11ea-931c-d94a08ec854e.png)

Look into the similar issues #390  and #627 .

**Comment by tikhsuP** [comment]
An easy fix is replacing the word 'Membership' by 'Join' as done in the Donate section above.

**Comment by Chaitanyassr** [comment]
> An easy fix is replacing the word 'Membership' by 'Join' as done in the Donate section above.

go ahead! make a PR :)

**Comment by tikhsuP** [comment]
@Chaitanyassr 
Kindly review.

**Comment by Chaitanyassr** [comment]
> An easy fix is replacing the word 'Membership' by 'Join' as done in the Donate section above.

you can reduce the size of membership in the respective css or make it responsive.....for more information studies the PR maid eariler.

**Comment by tikhsuP** [comment]
Yes, I can definitely do that but, that will affect the 'Donate' section too.
Should I do that? @Chaitanyassr
